### DISCUSSION TITLE
[Help Wanted] Self-Hosted setup v2:  Docker using Ubuntu 22.04.2 LTS having issues getting to complete start and no error log to view

### DISCUSSION THREAD

**Discussion by 07Romeo** [standardnotes/forum#3142] [General]
**Describe the issue**
When attempting to docker compose pull and bring up.  The session fails with the following error (attached).  I am new to Docker so this could be me still learning docker.

**To Reproduce**
![Screenshot from 2023-03-08 23-14-12](https://user-images.githubusercontent.com/73923071/223949599-bfb37a44-00bd-4754-b6ba-cd04d0a5fe3d.png)

Follow this install document through step 7
https://standardnotes.com/help/self-hosting/docker

Step errors and leaves no log in directory on step7

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] : Ubuntu 22.04.2 LTS running as a VM in the current version of TrueNAS Scale 22.12.1
 - Docker version: Docker version 23.0.1, build a5ee5b1
 - Browser [e.g. chrome, safari]  Firefox 
 - Version [e.g. 22] 110.0.1

N/A
**Smartphone (please complete the following information):**

**Additional context**

**Comment by jackyzy823** [comment]
your second part of command lacks `sudo`
since image is pulled , you only need to do `sudo docker compose up -d`
